src/main/java/org/apache/accumulo/access/AccessExpressionImpl.java

```diff
@@ -60,7 +55,7 @@ public Authorizations getAuthorizations() {
   }
 
   /**
-   * Creates a column visibility for a Mutation.
+   * Creates an AccessExpression for a Mutation.
```

Comment:
```suggestion
   * Creates an AccessExpression.
```

I don't know that we need to reference Mutation anywhere.